README.md

```diff
@@ -55,7 +55,8 @@ suspend fun main() = coroutineScope {
 * [ui](ui/README.md) &mdash; modules that provide coroutine dispatchers for various single-threaded UI libraries:
   * Android, JavaFX, and Swing.
 * [integration](integration/README.md) &mdash; modules that provide integration with various asynchronous callback- and future-based libraries:
-  * JDK8 [CompletionStage.await], Guava [ListenableFuture.await], and Google Play Services [Task.await];
+  * JDK8 [CompletionStage.await], Guava [ListenableFuture.await], Google Play Services [Task.await] and Huawei Mobile Services [Task.await](https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-huawei-services/kotlinx.coroutines.tasks/com.google.android.gms.tasks.-task/await.html
```

Comment:
I tried to include docs link by a reference (the same way as others), but `knit` tool was removing it. Do you know why?